### PR TITLE
release-2022-08-14

### PR DESCRIPTION
**Pull request by olsenme**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,45 @@
...
+## Release 2022-08-14
+
+Monitor the release status by regions at [AKS-Release-Tracker](http://aka.ms/aks/release-tracker).
+
+### Announcements
+
+* Starting with Kubernetes 1.25, the following changes will be made default:
+  * Ubuntu 22.04 for x86, AMD and ARM64 architectures will be the default host.
+  * Windows Server 2022 will be the default Windows host. Important, old windows 2019 containers will not work on windows server 2022 hosts.
+* Starting with Kubernetes 1.24, the following changes will be made default:
```
Comment (palma21) [inline CHANGELOG.md:12]: No longer needed?

Comment (palma21) [reply CHANGELOG.md:12]: the 1.24 section I mean

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,45 @@
...
+  * The [NodeRestriction Admission Controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#noderestriction) will be enabled.  This will allow users to enable/disable node restriction.
+  * CoreDNS version 1.9.2 will be default version. With this new version of CoreDNS wildcard queries are no longer allowed.
+  * metrics-server version 0.6.1 will be the default version.
+  * metrics-server vertical pod autoscaler will be enabled.
+* Kubernetes 1.21 version deprecation will start taking effect from July 31st, 2022.
+* Update your AKS labels to the recommended substitutions before deprecation after the Kubernetes v1.24 release. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Konnectivity rollout is finished in global and started in Sovereign (China, USGov).
```
Comment (palma21) [inline CHANGELOG.md:20]: Needs update/removal

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,45 @@
...
+* Docker will no longer be supported as a container runtime on Windows after September 1, 2022. Follow these [steps](https://docs.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli#:~:text=Upgrade%20an%20existing%20Windows%20Server%20node%20pool%20to%20containerd) in our documentation to upgrade your Kubernetes cluster to change your container runtime to containerd.
+
+### Release notes
+
+* Behavioral Changes
+  * Remove responseObject from kube-audit logs when its size is reaching log analytics column size limit (32K) and customer enables kube-audit/kube-audit-admin diagnostics.
+* Bug fixes
+       * Fix bug in processing fractional memory limits on Windows Nodes
```
Comment (palma21) [inline CHANGELOG.md:28]: too much indentation

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,45 @@
...
+       * Fix log loss due to inode reuse on Windows Nodes
+       * Fix issue with cert rotation on Windows nodes that caused VMSS inconsistency
+* Component Updates
+  * Windows
+    * Update to Ruby 3.1.1
+    * Update telegraf to 1.23.2
+    * Update fluentd to 1.14.6
+    * Use default JSON gem instead of yajl-json
+    * Consume tomlrb as a gem instead of committed source code
+    * Move from beta.kubernetes.io to kubernetes.io
```
Comment (palma21) [inline CHANGELOG.md:38]: What does this mean?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,45 @@
...
+
+* Behavioral Changes
+  * Remove responseObject from kube-audit logs when its size is reaching log analytics column size limit (32K) and customer enables kube-audit/kube-audit-admin diagnostics.
+* Bug fixes
+       * Fix bug in processing fractional memory limits on Windows Nodes
+       * Fix log loss due to inode reuse on Windows Nodes
+       * Fix issue with cert rotation on Windows nodes that caused VMSS inconsistency
+* Component Updates
+  * Windows
+    * Update to Ruby 3.1.1
```
Comment (palma21) [inline CHANGELOG.md:33]: what is this?

Comment (palma21) [reply CHANGELOG.md:33]: Is this in the windows VHD image?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,45 @@
...
+    * Update telegraf to 1.23.2
+    * Update fluentd to 1.14.6
+    * Use default JSON gem instead of yajl-json
+    * Consume tomlrb as a gem instead of committed source code
```
Comment (palma21) [inline CHANGELOG.md:37]: Why is this relevant

File: CHANGELOG.md (hunk 2 above)
Comment (olsenme) [reply CHANGELOG.md:20]: Just taking the 1.21 deprecation date out?

Comment (palma21) [reply CHANGELOG.md:13]: No, konnectivity.

**Review by kaarthis — APPROVED** [review]

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,33 @@
...
+  * Windows Server 2022 will be the default Windows host. Important, old windows 2019 containers will not work on windows server 2022 hosts.
+* Kubernetes 1.21 version deprecation will start taking effect from July 31st, 2022.
+* Update your AKS labels to the recommended substitutions before deprecation after the Kubernetes v1.24 release. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Docker will no longer be supported as a container runtime on Windows after September 1, 2022. Follow these [steps](https://docs.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli#:~:text=Upgrade%20an%20existing%20Windows%20Server%20node%20pool%20to%20containerd) in our documentation to upgrade your Kubernetes cluster to change your container runtime to containerd.
+* Removed `Microsoft.Resources/deployments/write`, `Microsoft.Insights/alertRules/*`, and `Microsoft.Support/*` from the [built-in Azure RBAC data plane roles for AKS](https://docs.microsoft.com/azure/aks/manage-azure-rbac#create-role-assignments-for-users-to-access-cluster).
```
Comment (shashankbarsin) [inline CHANGELOG.md:15]: this needs to be under bug fixes section under release notes below

**Review by shashankbarsin — APPROVED** [review]

**Pull request merged**